RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+State.swift

```diff
@@ -12,6 +13,43 @@ extension AccountPortfoliosClient {
 		public var debugDescription: String {
 			account.debugDescription
 		}
+
+		init(account: OnLedgerEntity.OnLedgerAccount, hiddenAssets: [AssetAddress]) {
+			var modified = account
+
+			// Remove every hidden fungible resource
+			modified.fungibleResources.nonXrdResources.removeAll(where: { resource in
+				hiddenAssets.contains(.fungible(resource.resourceAddress))
+			})
+
+			// Remove every hidden pool unit
+			modified.poolUnitResources.poolUnits.removeAll(where: { poolUnit in
+				hiddenAssets.contains(.poolUnit(poolUnit.resourcePoolAddress))
+			})
+
+			// We cannot remove the NFTs, since at this moment we haven't loaded them yet.
```

Comment:
In what sense are they not loaded yet? Also, this whole section confuses me, because I thought we only wanted to hide entire (non-fungible) resources, but perhaps this is older code that hasn't been removed yet?

Reply:
correct, we should hide the entire non fungible resource, not the specific non fungible ids.

Reply:
Yes, these are the changes that I am working on since the change on the designs after Friday's demo